scripts/us_bls/jolts/mcf_config.py

```diff
@@ -0,0 +1,674 @@
+_mcf_map = {
```

Comment:
Since this is the same as map_config.py, can't we just have one of these dicts?

Having two identical dicts makes maintenance long term harder as these two will have to be kept in sync for edits.